lib/rest-builder.js

```diff
@@ -481,13 +458,13 @@ RequestBuilder.prototype.invoke = function(parameters, cb) {
       delete json.query[q];
     }
   }
-  /* eslint-disable one-var */
+
   for (var h in json.headers) {
```

Comment:
Why wouldn't it work? the scope of the variable is applicable only within the brackets no? The reason I suggested that is because you removed the line to disable eslint's check for one character variables (I'm assuming that's what it means).